routes/curations.js

```diff
@@ -105,6 +105,11 @@ router.post('/reprocess', permissionsCheck('curate'), asyncMiddleware(reprocessM
 
 async function reprocessMergedCurations(request, respond) {
   const coordinatesArray = request.body.map(EntityCoordinates.fromString)
+  // Reprocess consume a lot of resource. Limit the size of the reprocess request.
+  const reprocessThreshold = 100;
+  if (coordinatesArray.length >= reprocessThreshold) {
```

Comment:
If it's over the threshold, can't we just cut it off at the threshold and reprocess what we can?

Reply:
We could do that too. But I think because we are using a job. It might be better to make it successful or not successful. Otherwise, your job needs to change for partial success case.

Reply:
Makes sense